src/modules/auth/resources/ResourcesDeletedListContainer.js

```diff
@@ -0,0 +1,161 @@
+import React, { useState, useEffect, useRef } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useSelector } from 'react-redux';
+
+import { DataTable, ButtonWithAuthz } from 'asab-webui';
+import { Container, Button } from 'reactstrap';
+
+const ResourcesDeletedListContainer = (props) => {
+
+	let SeaCatAuthAPI = props.app.axiosCreate('seacat_auth');
+
+	const [resources, setResources] = useState([]);
+	const [count, setCount] = useState(0);
+	const [page, setPage] = useState(1);
+	const [loading, setLoading] = useState(true);
+	const [show, setShow] = useState(false);
+	const [limit, setLimit] = useState(0);
+	const [height, setHeight] = useState(0);
+	const ref = useRef(null);
+	const { t } = useTranslation();
+
+	const retrieveButtonResource = "authz:tenant:admin";
+
+	const credentialsResources = useSelector(state => state.auth?.resources);
+
+	const headers = [
+		{
+			name: t("Name"),
+			key: '_id',
+			link: {
+				pathname: '/auth/resources/deleted/',
+				key: '_id'
+			}
+		},
+		{
+			name: t("Created at"),
+			key: '_c',
+			datetime: true
+		},
+		{
+			name: t("Description"),
+			key: 'description'
+		},
+		{
+			name: " ",
+			customComponent: {
+				generate: (resource) => (
+					<div className="d-flex justify-content-end">
+						<ButtonWithAuthz
+							title={t("ResourcesDeletedListContainer|Retrieve resource")}
+							id={resource._id}
+							size="sm"
+							color="primary"
+							outline
+							onClick={() => {confirmationPrompt(resource._id)}}
+							resource={retrieveButtonResource}
+							resources={credentialsResources}
+						>
+							<i className="cil-action-undo"></i>
+						</ButtonWithAuthz>
+					</div>
+				)
+			}
+		}
+	];
+
+	useEffect(() => {
+		setHeight(ref.current.clientHeight);
+	}, []);
+
+	useEffect(()=>{
+		setShow(false);
+		if (resources.length === 0) {
+			// Timeout delays appearance of content loader in DataTable. This measure prevents 'flickering effect' during fast fetch of data, where content loader appears just for a split second.
+			setTimeout(() => setShow(true), 500);
+		};
+		if (limit > 0) {
+			getResources();
+		}
+	}, [page, limit]);
+
+	// Fetches deleted resources
+	const getResources = async () => {
+		try {
+			let response = await SeaCatAuthAPI.get(`/resource`, {params: {p:page, i:limit, include_deleted:true}});
+			let deletedResources = [];
+			response.data.data.map((item) => {
```

Comment:
@petrKavulok Not sure if `response.data.data` is a correct way for GET API action. Can you please confirm with @byewokko ? In my point of view it should be just `response.data` since for GET calls you dont need to have a `result` so `data.data` are unnecessary here. Ofcourse if there is not anything else behind this

Reply:
`response.data` also contains `count`. so it makes sense to keep the actual list of resources in `response.data.data`.

Reply:
@Pe5h4 

<img width="772" alt="Screenshot 2023-03-07 at 16 14 01" src="https://user-images.githubusercontent.com/79644454/223465340-7b39ac98-4762-4c19-bc40-9e9b4c793419.png">

Reply:
Okok